stm32 : power_mgmt : u0 : update dts property and add overlay for  blinky_pm test

djiatsaf-st opened the pull request:
- Define stm32_lp_tick_source for STM32U0_DK board since PM is looking for this node and change clock source for LPTIM2 to LSI.

- Also disable LPTIM2 by default to avoid build failures caused by enabling multiple LPTIM peripherals. 

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/92300

djiatsaf-st committed:
samples: boards: st: power_mgmt: blinky: add stm32u083_dk board overlay

define stm32_lp_tick_source since PM is looking for this node
and change clock source for lptim2 to lsi.

Signed-off-by: Fabrice DJIATSA <fabrice.djiatsa-ext@st.com>

erwango reviewed (commented):
Why not putting this directly in board ?

djiatsaf-st committed:
dts: arm: st: u0: disable lptim2 by default

LPTIM2 is enabled by default; disable it to avoid
the build failure:
"Error: Only one LPTIM instance should be enabled"
when LPTIM1 is also enabled in the DTS.

Signed-off-by: Fabrice DJIATSA <fabrice.djiatsa-ext@st.com>

FRASTM commented on boards/st/stm32u083c_dk/stm32u083c_dk.dts:
```diff
@@ -134,6 +134,12 @@
 };
 
+stm32_lp_tick_source: &lptim2 {
+	clocks = <&rcc STM32_CLOCK(APB1, 30)>,
```
the stm32u0.dtsi has 
  clocks = <&rcc STM32_CLOCK(APB1, 30**U**)>;  
same here ?
